src/opensearch.py

```diff
@@ -35,21 +35,61 @@
 logger = logging.getLogger(__name__)
 
 
+BASE_SNAP_DIR = "/var/snap/opensearch"
+SNAP_DATA = f"{BASE_SNAP_DIR}/current"
+SNAP_COMMON = f"{BASE_SNAP_DIR}/common"
+SNAP = "/snap/opensearch/current"
+
+
+OPENSEARCH_SNAP_METADATA = {
```

Comment:
Is this needed? we do not support upgrades on 2.11

Reply:
@carlcsaposs-canonical @Mehdi-Bendriss that is not what we have right now....

And deciding to support "2.11.1" or has nothing to do with `OPENSEARCH_SNAP_METADATA`. We will certainly face a JDK v21 > JDK v22 at some point; and OpenSearch community does these moves within minor versions.